src/main/java/org/opensearch/performanceanalyzer/http_action/config/RestConfig.java

```diff
@@ -8,5 +8,5 @@
 /** Shared constants and configuration used by the REST controllers defined by this plugin. */
 public class RestConfig {
     public static final String PA_BASE_URI = "/_plugins/_performanceanalyzer";
-    public static final String LEGACY_PA_BASE_URI = "/_opendistro/_performanceanalyzer";
+    public static final String LEGACY_PA_BASE_URI = PA_BASE_URI;
```

Comment:
We can remove this check. As legacy _opendistro API's are no more supported.

Reply:
Reverted back